spec/index/index_spec.rb

```diff
@@ -70,6 +70,40 @@
     end
   end
 
+  context '#sort' do
+    let(:idx1) { Daru::Index.new ['speaker', 'mic', 'guitar', 'amp'] }
+    let(:idx2) { Daru::Index.new [:one, 'one', 1, 2, :two] }
```

Comment:
Thanks @zverok @v0dro , I have changed these line in[ new commit](https://github.com/SciRuby/daru/pull/315/commits/a717aa0698dba7c5286bcc73ab2d31fbf4437753)